### PR TITLE
feat(pie-icons-webc): DSW-1543 update README with Vue documentation

### PR DESCRIPTION
**Pull request by LTurns**
## Describe your changes (can list changeset entries if preferable)
[Added] - Documentation for usage inside Vue app 
(tested in Vue-3 example app and Nuxt in pie-aperture) 

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests
- [x] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [x] If it is a component change, I have reviewed the Storybook preview
- [x] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [x] If it is a `PIE Docs` change, I have reviewed the PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2
- [x] If it is a `PIE Docs` change, I have reviewed the PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by jamieomaguire — DISMISSED** [review]

File: packages/tools/pie-icons-webc/README.md [hunk 1]
```diff
@@ -129,9 +129,24 @@ export default function App() {
...
+To import from the package root:
+
+```tsx
+import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-webc";
+
+export default function App() {
+  return (
+    <div className="App">
+      <icon-calendar />
+      <icon-alert-triangle-large />
```
Comment (xander-marjoram) [inline packages/tools/pie-icons-webc/README.md:145]: I don't think custom elements are allowed to be self-closing yet
```suggestion
      <icon-calendar></icon-calendar>
      <icon-alert-triangle-large></icon-alert-triangle-large>
```

Comment (LTurns) [reply packages/tools/pie-icons-webc/README.md:145]: Self-closing actually worked for me in both examples so maybe this is allowed now? 🎉

Comment (LTurns) [reply packages/tools/pie-icons-webc/README.md:145]: spoke to @xander-marjoram, agreed to leave for now 👍

Comment (jamieomaguire) [reply packages/tools/pie-icons-webc/README.md:145]: I would suggest indeed self-closing the tags. The reason for this is custom elements are not void elements (they can content children).

I would be interested in seeing how they worked for you @LTurns but I would still suggest following best practice in the docs here

**Review by jamieomaguire — APPROVED** [review]

**Review by raoufswe — APPROVED** [review]

**Pull request merged**